tracing-opentelemetry/src/subscriber.rs

```diff
@@ -197,6 +212,13 @@ impl<'a> field::Visit for SpanAttributeVisitor<'a> {
         self.record(KeyValue::new(field.name(), value));
     }
 
+    /// Set attributes on the underlying OpenTelemetry [`Span`] from `i64` values.
```

Comment:
i think this should say
```suggestion
    /// Set attributes on the underlying OpenTelemetry [`Span`] from `f64` values.
```